.github/workflows/tests.yml

```diff
@@ -29,7 +29,7 @@ defaults:
 
 jobs:
   build:
-
+    if: github.repository_owner=='tardis-sn'
```

Comment:
Leave spaces before and after ==

Reply:
Done via https://github.com/tardis-sn/tardis/pull/2280/commits/8351415447c05920aa85e0d657e4ba811dc20528